test/testnet_scion/README.md

```diff
@@ -88,7 +88,7 @@ E.g., with `sudo -u $USER ...` or by using `nsenter` with `setuid` and `setguid`
 
 
 ## LightningFilter
-To run LF in the corresponding namespaces the script ``run_lf.sh`` can be used.
+To run LF in the corresponding namespaces the script ``run_lf.sh`` can be used. NOTE that your system should have at least 4 cores s.t. each LF can have its own core and you can still perform pings. If the logs say that there are no cores available then this might be the issue.
```

Comment:
Have you experienced performance issues?
Potentially running both LFs on the same core, leaving the other core(s) to the system, could also work.

```suggestion
To run LF in the corresponding namespaces the script ``run_lf.sh`` can be used.

> NOTE that your system should have at least 4 cores s.t. each LF can have its own core and you can still perform pings. If the logs say that there are no cores available then this might be the issue.
```

Reply:
I tested with 1, 2, and 4 cores (available to VirtualBox VM). When running with only 1 core I get the following error message in lf1.log.


```
LF INFO: 0.1.0- Git: bbf4418b
EAL: Detected CPU lcores: 1
EAL: Detected NUMA nodes: 1
EAL: core 1 unavailable
EAL: invalid parameter for --lcores
```


When running with 2 cores I had performance issues and about 95% packet loss for just the scion pings. 

4 cores worked fine, thats why I chose to add it to the documentation.

Reply:
Thank you for the elaboration.
Yes, the current configuration requires at least 2 cores. But having 95% of packets lost is not feasible and this comment can be really helpful.